### PR TITLE
Clean up the about package a bit

### PR DESCRIPTION
**Pull request by TacoTheDank**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [x] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
I looked at my PR from when I converted the about package to Kotlin and found some unsatisfactory things.

- Grouped the page titles back into the state adapter because it makes more sense for it to be there.
- Removed an unneeded JvmStatic annotation.
- Extracted some duplicate code.

#### APK testing 
The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.kt [hunk 1]
```diff
@@ -87,61 +88,57 @@ object LicenseFragmentHelper {
...
+                        }
+                        setNeutralButton(R.string.open_website_license) { _, _ ->
+                            ShareUtils.openUrlInBrowser(context, component.link)
+                        }
+                        show()
                     }
-                    alert.show()
                 }
         }
     }
```
Comment (litetex) [inline app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.kt:134]: These methods are nearly identical.
Can they be merged into one method?

(also then we wouldn't need ``loadLicense`` below)

**Review by litetex — COMMENTED** [review]
LGTM

one minor thing:

File: app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.kt (hunk 1 above)
Comment (TacoTheDank) [reply app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.kt:134]: Uh, perhaps? I'll see what I can do.

Comment (TacoTheDank) [reply app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.kt]: Update: Kotlin is a godsend.

**Review by litetex — APPROVED** [review]
LGTM

No problems when testing the code

**Pull request merged**